README.rst

```diff
@@ -75,15 +75,7 @@ Installation
 
 or
 
-::
-
-    pip install git+https://github.com/gleitz/howdoi.git#egg=howdoi
-
-or
-
-::
-
-    python setup.py install
+Check out howdoi docs for more ways of installation.
```

Comment:
Could we link this to http://gleitz.github.io/howdoi/installation/ ?

Reply:
I was hoping once mkdocs get merged, I would go about editing readme just for adding links since I have changes many things in mkdocs and many things could be back linked. But for backlinking , I would need the built and deployed mkdocs link.

Reply:
Howdoi installation was removed from mkdocs and the introduction page had both introduction and installation.